src/wp-includes/theme.php

```diff
@@ -4181,7 +4181,7 @@ function _add_default_theme_supports() {
 	add_theme_support( 'post-thumbnails' );
 	add_theme_support( 'responsive-embeds' );
 	add_theme_support( 'editor-styles' );
-	add_theme_support( 'html5', array( 'comment-form', 'comment-list', 'style', 'script' ) );
+	add_theme_support( 'html5', array( 'comment-form', 'comment-list', 'search-form', 'gallery', 'caption', 'style', 'script' ) );
```

Comment:
Let's add a comment here to clearly identify that this line of code makes block themes support HTML5 markup by default.

```suggestion
	/*
	 * Make block themes support HTML5 by default for all non-block content.
	 * Blocks contain their own markup, which is HTML5.
	 */
	add_theme_support( 'html5', array( 'comment-form', 'comment-list', 'search-form', 'gallery', 'caption', 'style', 'script' ) );
```